lib/routes/malaysiakini/index.ts

```diff
@@ -0,0 +1,119 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import parser from '@/utils/rss-parser';
+import { config } from '@/config';
+
+export const route: Route = {
+    path: '/:lang/:category?',
+    categories: ['new-media'],
+    example: '/malaysiakini',
```

Comment:
Since `lang` is a compulsory parameter, it's expected that the example also include this parameter